src/toil/test/docs/scripts/tutorial_discoverfiles.py

```diff
@@ -0,0 +1,33 @@
+import subprocess
+import os
+from toil.common import Toil
+from toil.job import Job
+
+class discoverFiles(Job):
+    """Views files at a specified path using ls."""
+    def __init__(self, path, *args, **kwargs):
+        self.path = path
+        super(discoverFiles, self).__init__(*args, **kwargs)
+
+    def run(self, fileStore):
+        subprocess.check_call(["ls", self.path])
```

Comment:
The statement below `job1 = discoverFiles(path="/System/", displayName='sysFiles')` is only true for Apple and will be `job1 = discoverFiles(path="/sys/", displayName='sysFiles')` on Linux.  The easier fix would be to add a statement here in the run function like:

```
if os.path.exists(self.path):
    subprocess.check_call(["ls", self.path])
```

To prevent it from erroring on a path that does not exist on a given system.